src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1680,10 +1684,11 @@ void CodeGen::genSSE2Intrinsic(GenTreeHWIntrinsic* node)
             assert(op1 != nullptr);
             assert(op2 != nullptr);
 
-            op2Reg          = op2->gtRegNum;
-            instruction ins = HWIntrinsicInfo::lookupIns(intrinsicId, baseType);
-            op1Reg          = op1->gtRegNum;
-            emit->emitIns_AR_R(ins, emitTypeSize(baseType), op2Reg, op1Reg, 0);
+            op2Reg                = op2->gtRegNum;
+            instruction ins       = HWIntrinsicInfo::lookupIns(intrinsicId, baseType);
+            op1Reg                = op1->gtRegNum;
+            GenTreeStoreInd store = storeIndirForm(node->TypeGet(), op1, op2);
```

Comment:
You no longer need to set `op1Reg` or `op2Reg` here ?

Reply:
Good catch; fixed.